api/handler_test.go

```diff
@@ -86,7 +86,7 @@ func getHandler(t *testing.T, opts ...testutil.GetDBOption) (http.Handler, *depe
 	})
 	meta := auth.NewDBMetadataManager("dev", conn)
 	retentionService := retention.NewService(conn)
-	migrator := db.NewDatabaseMigrator(dbparams.Database{URI: handlerDatabaseURI})
+	migrator := db.NewDatabaseMigrator(dbparams.Database{ConnectionString: handlerDatabaseURI})
```

Comment:
I remain confused about whether this is intended to be used as a URI or as a connection string.  I *think* it is a connection string, so we should call it that throughout the code and in YaML config etc.

Reply:
This could also be in a later PR, of course, if we want to get this one of our plates.

Reply:
The configuration and code are now aligned - connection_string (yaml) and ConnectionString (go)